CONTRIBUTING.md

```diff
@@ -16,68 +16,83 @@ This is the best way to contribute to jQuery UI. Please read through the full gu
 
 Every week (unless otherwise noted) the jQuery UI team has a meeting to discuss the progress of current work and to bring forward possible new blockers for discussion. The meeting is held on [IRC](http://irc.jquery.org) in the #jquery-meeting channel at [Noon EST](http://www.timeanddate.com/worldclock/fixedtime.html?month=1&day=17&year=2011&hour=12&min=0&sec=0&p1=43) on Wednesdays. Meeting notes are posted on http://meetings.jquery.org/category/ui/ after each meeting.
 
-## Tips For Bug Patching
+## Tips for Getting Started
 
-### Environment: localhost w/ PHP, Node.js & Grunt
+### Environment: Minimum Required
 
-jQuery UI uses Node.js & Grunt to automate the building and validation of source code.
+If you are contributing changes you will need a fork of jquery (see [Getting the Source](#environment-getting-the-source)). If you just want the source code you could clone jquery ui directly:
```

Comment:
"clone jquery ui directly"

It's always either jquery-ui (the repo) or jQuery UI (the project).